audeer/core/io.py

```diff
@@ -458,6 +458,33 @@ def replace_file_extension(
     return f'{path[:-len(extension)]}{new_extension}'
 
 
+def rmdir(
+        path: typing.Union[str, bytes],
+):
+    """Remove directory.
+
+    Remove the directory
+    and all its content
+    if the directory exists.
+
+    Args:
+        path: absolute or relative path of directory to remove
+
+    Raises:
+        NotADirectoryError: if path is not a directory
+
+    Example:
+        >>> p = mkdir('path1/path2/path3')
```

Comment:
why assigning to `p`?

Reply:
I assign to a variable as otherwise the test will fail.
So maybe better
```suggestion
        >>> _ = mkdir('path1/path2/path3')
```
?

Reply:
At the beginning I thought about reusing the returned path, but `os.path.dirname(p)` is much harder to understand than just `path1/path2`

Reply:
Alternatively you could use `# doctest: +SKIP`, but I'm also fine with your solution.

Reply:
Yeah, good suggestion as we test this line anyway by the next lines and it looks cleaner without the return variable.
I changed to use `# doctest: +SKIP`